sdk/keyvault/azure-keyvault-administration/tests/perfstress_tests/get_role_definition.py

```diff
@@ -0,0 +1,57 @@
+# ------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ------------------------------------
+
+from azure_devtools.perfstress_tests import PerfStressTest
+from azure.identity import DefaultAzureCredential
+from azure.identity.aio import DefaultAzureCredential as AsyncDefaultAzureCredential
+from azure.keyvault.administration import (
+    KeyVaultAccessControlClient, 
+    KeyVaultDataAction,
+    KeyVaultPermission,
+     KeyVaultRoleScope,
+)
+from azure.keyvault.administration.aio import KeyVaultAccessControlClient as AsyncKeyVaultAccessControlClient
+
+
+class GetRoleDefinitionTest(PerfStressTest):
+
+    def __init__(self, arguments):
+        super().__init__(arguments)
+
+        # Auth configuration
+        self.credential = DefaultAzureCredential()
+        self.async_credential = AsyncDefaultAzureCredential()
+
+        # Create clients
+        vault_url = self.get_from_env("AZURE_KEYVAULT_URL")
+        self.client = KeyVaultAccessControlClient(vault_url, self.credential, **self._client_kwargs)
+        self.async_client = AsyncKeyVaultAccessControlClient(vault_url, self.async_credential, **self._client_kwargs)
+        self.role_name = "livekvtestgetroledefperfadmin"
```

Comment:
The role definition name has to be a GUID, so we can `import uuid` and self `self.role_name = uuid.uuid4()`